src/iap/shared/components/carousel/carousel.component.scss

```
@@ -0,0 +1,94 @@
+@import '/src/scss/constants.scss';
+@import '/src/scss/mixins.scss';
+
+$button-margin: 50px;
```

:point_right: :point_left: 
```suggestion
$iap-button-margin: 50px;
```